### PR TITLE
Adding to faqs

### PR DESCRIPTION
**Pull request by anirudTT**
Additions in this PR:

- Adds more information in the FAQs on how a user can perform software resets 
- added direct links to the tt-smi repo within the FAQ
- Provides a additional clarification in the [first_5_steps/1_install_tt_buda.md](https://github.com/tenstorrent/tt-buda-demos/compare/anirud/adding-to-faqs?expand=1#:~:text=Please%20navigate%20to%20%5Btt%2Dflash,Firmware%2520files%2Cof%2520the%2520images) during the install of tt-flash and tt-firmware-gs

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -2,13 +2,20 @@
 
 ## Resetting an accelerator board
 
-If a Tenstorrent chip seems to hang and/or is producing unexpected behaviour,
-you may try a software reset of the board.
+If you encounter a situation where a Tenstorrent chip appears to be unresponsive or is exhibiting unusual behavior, a software reset of the board might be a viable solution.
 
 For Grayskull: `tt-smi -tr all`
```
Comment (milank94) [inline FAQ.md:7]: Remove this line. It's redundant if you are mentioning `tt-smi -lr 0`.

File: FAQ.md [hunk 2]
```diff
@@ -2,13 +2,20 @@
...
 
-If the software reset does not work, unfortunately you will have to power cycle
-the board. This usually means rebooting the host of a board.
+For a software reset on a single chip use : `tt-smi -lr 0` .
+
+For more information on performing reset on multiple chips or other specifics visit this [link](https://github.com/tenstorrent/tt-smi?tab=readme-ov-file#resets:~:text=on%20the%20footer.-,Resets,-Another%20feature%20of)
```
Comment (milank94) [inline FAQ.md:11]: Specify where the link leads to

```suggestion
For more information on performing reset on multiple chips or other specifics visit [TT-SMI homepage](https://github.com/tenstorrent/tt-smi?tab=readme-ov-file#resets:~:text=on%20the%20footer.-,Resets,-Another%20feature%20of)
```

File: first_5_steps/1_install_tt_buda.md [hunk 3]
```diff
@@ -58,6 +58,7 @@ Please navigate to [tt-kmd](https://github.com/tenstorrent/tt-kmd).
 ### Device Firmware Update
 
 Please navigate to [tt-flash](https://github.com/tenstorrent/tt-flash) and [tt-firmware-gs](https://github.com/tenstorrent/tt-firmware-gs).
+The [tt-firmware-gs](https://github.com/tenstorrent/tt-firmware-gs) firmware file needs to be installed using the [tt-flash](https://github.com/tenstorrent/tt-flash) utility, for more details visit this [link](https://github.com/tenstorrent/tt-flash?tab=readme-ov-file#firmware-files:~:text=Firmware%20files,of%20the%20images.)
```
Comment (milank94) [inline first_5_steps/1_install_tt_buda.md:61]: Add something more descriptive for the link.

```suggestion
The [tt-firmware-gs](https://github.com/tenstorrent/tt-firmware-gs) firmware file needs to be installed using the [tt-flash](https://github.com/tenstorrent/tt-flash) utility, for more details visit [TT-Flash homepage](https://github.com/tenstorrent/tt-flash?tab=readme-ov-file#firmware-files:~:text=Firmware%20files,of%20the%20images.)
```

File: first_5_steps/1_install_tt_buda.md [hunk 4]
```diff
@@ -58,6 +58,7 @@ Please navigate to [tt-kmd](https://github.com/tenstorrent/tt-kmd).
 ### Device Firmware Update
 
 Please navigate to [tt-flash](https://github.com/tenstorrent/tt-flash) and [tt-firmware-gs](https://github.com/tenstorrent/tt-firmware-gs).
```
Comment (milank94) [inline first_5_steps/1_install_tt_buda.md:60]: Remove this line. It's redundant with the next one.

**Comment by anirudTT** [comment]
test

File: first_5_steps/1_install_tt_buda.md (hunk 4 above)
Comment (anirudTT) [reply first_5_steps/1_install_tt_buda.md:60]: will do this

File: FAQ.md (hunk 1 above)
Comment (anirudTT) [reply FAQ.md:7]: removing

File: first_5_steps/1_install_tt_buda.md (hunk 4 above)
Comment (anirudTT) [reply first_5_steps/1_install_tt_buda.md]: done

File: first_5_steps/2_running_nlp_models.ipynb
[the remark this answers is not in the document]
Comment (milank94) [reply first_5_steps/2_running_nlp_models.ipynb:1]: Still seeing this file in the proposed changes. Try to see if you can revert any of the changes done to it.

File: FAQ.md [hunk 5]
```diff
@@ -2,13 +2,19 @@
...
 
-If the software reset does not work, unfortunately you will have to power cycle
-the board. This usually means rebooting the host of a board.
+For a software reset on a single chip use : `tt-smi -lr 0` .
+
+For more information on performing reset on multiple chips or other specifics visit [TT-SMI homepage](https://github.com/tenstorrent/tt-smi?tab=readme-ov-file#resets:~:text=on%20the%20footer.-,Resets,-Another%20feature%20of)
+
+If you need additional assistance, you can access a detailed explanation of all available command options by appending the help flag to the command like so: `tt-smi --help` or `tt-smi -h`.
+
+For comprehensive insights and detailed instructions on utilizing the command line GUI, we invite you to explore the Tenstorrent System Management Interface (TT-SMI) repository on GitHub at [tt-smi-repo](https://github.com/tenstorrent/tt-smi). TT-SMI serves as a versatile command-line utility tailored to streamline interaction with all Tenstorrent devices on host.
```
Comment (milank94) [inline FAQ.md:14]: Nit-picking the links.

```suggestion
For comprehensive insights and detailed instructions on utilizing the command line GUI, we invite you to explore the Tenstorrent System Management Interface (TT-SMI) repository on GitHub at [TT-SMI homepage](https://github.com/tenstorrent/tt-smi). TT-SMI serves as a versatile command-line utility tailored to streamline interaction with all Tenstorrent devices on host.
```

File: FAQ.md [hunk 6]
```diff
@@ -2,13 +2,19 @@
...
-you may try a software reset of the board.
+If you encounter a situation where a Tenstorrent chip appears to be unresponsive or is exhibiting unusual behavior, a software reset of the board might be a viable solution.
 
-For Grayskull: `tt-smi -tr all`
 
-If the software reset does not work, unfortunately you will have to power cycle
-the board. This usually means rebooting the host of a board.
+For a software reset on a single chip use : `tt-smi -lr 0` .
+
+For more information on performing reset on multiple chips or other specifics visit [TT-SMI homepage](https://github.com/tenstorrent/tt-smi?tab=readme-ov-file#resets:~:text=on%20the%20footer.-,Resets,-Another%20feature%20of)
```
Comment (milank94) [inline FAQ.md:10]:
```suggestion
For more information on performing reset on multiple chips or other specifics visit [TT-SMI Resets](https://github.com/tenstorrent/tt-smi?tab=readme-ov-file#resets:~:text=on%20the%20footer.-,Resets,-Another%20feature%20of)
```

**Review by anirudTT — COMMENTED** [review]
- removed the metadata changes 
- incorporated the suggestions on the PR

**Review by anirudTT — COMMENTED** [review]
re corrected to remove metadata

**Review by milank94 — APPROVED** [review]
Looks good!

**Pull request merged**